README.md

```diff
@@ -41,9 +41,21 @@ Add "vtex.shelf" as your app dependency.
   - Default: 'Default Title'
 ```
 
+## CSS Classes
+
+```css
+vtex-shelf
+vtex-shelf__title-text
+vtex-shelf__title-content
+vtex-shelf__slide
+vtex-shelf__arrow-right
+vtex-shelf__arrow-left
+vtex-shelf__dots
```

Comment:
`vtex-shelf__arrow-right--disabled`
`vtex-shelf__arrow-left--disabled`

Reply:
@brunojdo the react-slick sets dynamically a class named 'slick-disabled' to when the arrow is disabled. If i want to modify the disable state of an arrow, the only way is to override the selector ```.vtex-shelf__arrow-right .slick-disabled```